GoogleTestAdapter/Tests.Common/ResultChecker/TrxResultChecker.cs

```diff
@@ -61,7 +61,8 @@ public void RunTestsAndCheckOutput(string typeName, string arguments, string tes
                 Assert.Inconclusive($"First time this test runs, created golden file - check for correctness! File: {goldenFile}");
             }
             CheckIfFileIsParsable(goldenFile);
-            if (!CompareXmlFiles(goldenFile, transformedResultFile, out string diffFile))
+            string diffFile;
```

Comment:
You can ignore all these commits, as they were just meant to support building GTA with V2015:
  |   | remove C# 7.0 features |   |   | 048822d
  |   | Install Microsoft.VSSDK.BuildTools depending on VS version |   |   | 979c4b2
  |   | Fix SampleTests (/DEBUG vs /DEBUG:FULL) |   |   | b9e1303
  |   | Comments for this crazy "install Microsoft.VSSDK.BuildTools" custom b… … |   |   | 19b24a5
  |   | try to get it right when VisualStudioVersion is not set |   |   | e5f0d85
  |   | Fix end-to-end tests for VS 2015. … |   |   | 1eefc9a